ci4/tests/unit/HealthTest.php

```
@@ -0,0 +1,49 @@
+    {
+        $this->assertTrue(defined('APPPATH'));
+    }
+
+    public function testBaseUrlHasBeenSet(): void
+    {
+        $validation = service('validation');
+
+        $env = false;
+
+        // Check the baseURL in .env
+        if (is_file(HOMEPATH . '.env')) {
```

PHPStan (2.2.2)

[error] Constant HOMEPATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)